src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
```

Comment:
```suggestion
  let price = days * 40;
```

40 is magic number here